neonvm/runner/main.go

```diff
@@ -574,6 +578,14 @@ func main() {
 	// disk details
 	qemuCmd = append(qemuCmd, "-drive", fmt.Sprintf("id=rootdisk,file=%s,if=virtio,media=disk,index=0,cache=none", rootDiskPath))
 	qemuCmd = append(qemuCmd, "-drive", fmt.Sprintf("id=runtime,file=%s,if=virtio,media=cdrom,readonly=on,cache=none", runtimeDiskPath))
+	{
+		diskName := "ssh-authorized-keys"
+		diskPath := fmt.Sprintf("%s/%s.qcow2", mountedDiskPath, "ssh-authorized-keys")
+		if err := createISO9660FromPath(logger, diskName, diskPath, "/vm/ssh"); err != nil {
+			logger.Fatal("Failed to create ISO9660 image", zap.Error(err))
```

Comment:
This crashes the program, no?

Reply:
Why?

Reply:
That's what https://pkg.go.dev/go.uber.org/zap#Logger.Fatal does.

Reply:
Yes that was intended. So you suggest just printing an error instead of exiting the program here?

Reply:
Yeah, programs should not exit from the middle of the stack. There might be defers, which should be executed, proper outer logging. We might at some point decide to recover from the error, e.g. skip ssh entirely.

Reply:
Currently we don't have defers in runner's main.go and we have used `log.Fatal` everywhere in there. The question is if we want to consider this error fatal or not. @sharnoff what do you think?

Reply:
Yeah, we're already using `log.Fatal` in a bunch of places. Practically speaking, it doesn't _really_ matter whether we do cleanup because the entire container gets cleaned up when `neonvm-runner` exits.

But this is still kind of "spooky action at a distance" and we should mark it as tech debt and plan to resolve sometime.

Reply:
Ah, that's `main()`, I see. Not used to code being in the `main()`. More used to smth like:
```
func run() error {
 ....
}

func main() {
  err := run()
  if err !=  nil {
     log.Error("error happened")
  }
}
```

The reason the code should be written like this is that when someone decides to move code somewhere else, like in a library, and there is no need to replaces `Fatal`s with returning error.

But since that's how code was before you, you can leave it as-is. 

> The question is if we want to consider this error fatal or not

I don't think it is very realistic this error happens, or even if it does, there is not much reason to run the VM without ssh. So it can be fatal.